index.html

```diff
@@ -1085,231 +1147,219 @@ <h2>ConsumedThing Examples</h2>
     <h2>The <dfn>InteractionData</dfn> interface</h2>
     <p>
       Belongs to the <a>WoT Consumer</a> conformance class.
-      Represents the data returned by <a>WoT Interactions</a> when no
-      explicit <a>DataSchema</a> is available for the <a>WoT Interaction</a>,
-      therefore applications need to rely on a <a href="https://www.w3.org/TR/2019/CR-wot-thing-description-20190516/#form">Form</a>'s `contentType`
-      defined for the <a>WoT Interaction</a>.
-      Implementations MAY always return an {{InteractionData}}
-      object for <a>WoT Interactions</a>, but MAY also parse the return data
-      according to the interaction's <a>DataSchema</a> when that is available.
+      Represents the data used by <a>WoT Interactions</a>.
     </p>
     <p class="ednote">
       Previously `any` has been used for representing data that
-      could be returned by <a>WoT Interactions</a>. Since data type can be
-      specified in the <a>TD</a> by a <a href="https://www.w3.org/TR/2019/CR-wot-thing-description-20190516/#form">Form</a>'s `contentType`
-      and optionally a <a>DataSchema</a>,
-      this interface enables handling returned data by applications.
+      could be returned by <a>WoT Interactions</a>. Data type is specified in
+      the <a>TD</a> by a <a href="https://www.w3.org/TR/2019/CR-wot-thing-description-20190516/#form">Form</a>'s `contentType`
+      and optionally a <a>DataSchema</a>.
+      This interface enables handling returned data by applications as a stream
+      or
```

Comment:
There seems to be something missing

Reply:
Yes, thanks, I fixed that (reformulated the whole part, please check if it's accurate).

Reply:
The description looks fine now. I still need to make up my mind and explore `InteractionData` with code.